File: C#/BluffinMuffin.Protocol/Lobby/RegisteredMode/GetUserResponse.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Lobby.RegisteredMode
{
    /// <summary>
    /// 
    /// </summary>
    public class GetUserResponse : AbstractBluffinReponse<GetUserCommand>
    {
        /// <summary>
        /// The email of the user
        /// </summary>
        [ExampleValue("ericmas001@hotmail.com")]
        public string Email { get; set; }
        /// <summary>
        /// The display name of the user
        /// </summary>
        [ExampleValue("Sponge Bob")]
        public string DisplayName { get; set; }
        /// <summary>
        /// The money currently in bank for this user
        /// </summary>
        [ExampleValue(42000.42)]
        public double Money { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public GetUserResponse(GetUserCommand command)
            : base(command)
        {
        }
    }
}

File: Python/BluffinMuffin.Protocol/lobby/registered_mode/get_user_response.py
from abstract_response import AbstractResponse
from lobby.registered_mode.authenticate_user_command import AuthenticateUserCommand
from lobby.registered_mode.check_display_exist_command import CheckDisplayExistCommand
from lobby.registered_mode.check_user_exist_command import CheckUserExistCommand
from lobby.registered_mode.get_user_command import GetUserCommand

__author__ = 'ericmas001@gmail.com'



class GetUserResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__(obj, GetUserCommand(obj['Command']))
        self.email = obj['Email']
        self.display_name = obj['DisplayName']
        self.money = obj['Money']

    def __str__( self ):
        return '{0} => ({1}, {2}, {3})'.format(super().__str__(), self.email, self.display_name, self.money)